packages/react-server/src/webpack-plugin/index.ts

```diff
@@ -0,0 +1 @@
+export {ReactServerPlugin} from './webpack-plugin';
```

Comment:
I think we need to add a `webpack-plugin.js` to the root of the package if we want to be able to get at this (see other packages with multiple sub-packages)

Reply:
its already included as `packages/react-server/webpack.d.ts` and `packages/react-server/webpack.js` and reference in `packages/react-server/package.json`

Reply:
thou I might rename them...